res/values-pt-rPT/strings.xml

```diff
@@ -784,7 +784,7 @@
     <string name="Failed_to_send_mail_">"Falha ao enviar e-mail."</string>
     <string name="In_Game_Mail">"Mail no Jogo"</string>
     <string name="Nav_Buttons_">"Botões de Nav.:"</string>
-    <string name="account_warning">"Nunca compartilha a tua palavra-passe da conta ou informações pessoais com ninguém por qualquer motivo!"</string>
+    <string name="account_warning">"Nunca partilha a tua palavra-passe da conta ou informações pessoais com ninguém por qualquer motivo!"</string>
```

Comment:
"Nunca compartilhes a tua palavra passe..." sounds better

Reply:
Oh, got it. I thought in Portugal you just use "partilhar" or something like that. Thanks for clarifying!